projects/client/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -1,43 +1,49 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
     <VersionPrefix>6.0.0</VersionPrefix>
```

Comment:
This should be removed.

Reply:
Yeah, we're using this as part of our CI scripts but I think that's going to change quite a bit. I'll remove it.